tools/mason/MasonUpdate.chpl

```diff
@@ -66,12 +66,13 @@ proc UpdateLock(args: list(string), tf="Mason.toml", lf="Mason.lock") {
     const tomlPath = projectHome + "/" + tf;
     const lockPath = projectHome + "/" + lf;
 
-
     updateRegistry(tf, args);
     const openFile = openreader(tomlPath);
     const TomlFile = parseToml(openFile);
+    if !isDir(SPACK_ROOT) || !isDir(MASON_HOME+'/spack-registry') then
+    throw new owned MasonError("Mason could not find Spack at $MASON_HOME. " +
+                  "To install Spack, use `mason external --setup`.");
```

Comment:
@ben-albrecht I think we should remove this because if an user doesn't want to use spack, and doesn't have spack installed, they might also see this message. Perhaps should first check if SPACK_ROOT and spack-registry is present, and then check if version is not updated and then show the error. What do u think?

Reply:
I agree. I think the `MasonBuild` check when `external` is present in the manifest is sufficient. Good catch!

Reply:
I've added a check in 48186a2. I believe that should be sufficient ?

Reply:
Yes, I think so. Thanks